tools/sigclient/README.md

```diff
@@ -34,6 +37,18 @@ Different Types of Readers:
 2. Dynamic User: Randomly Generates user events. These random events are generated using [gofakeit](github.com/brianvoe/gofakeit/v6).
 3. File: Reads a file line by line. Expects each line is a new json. Will loop over file if necessary
 
+To Ingest 200 columns per record:
+
+```bash
+go run main.go ingest esbulk -n 1 -g benchmark -d http://localhost:8081/elastic -t 1000 -p 1 --enableVariableNumColumns true --maxColumns 200
```

Comment:
ijn this case `minColumns` will default to 0, so would it generate between 0 and 200 num of columns per record ?

Reply:
If `minColumns` is 0, then that value will be set to the `maxColumns` value which is `200` in this case.